docs/releases/4.2.4.txt

```diff
@@ -9,4 +9,6 @@ Django 4.2.4 fixes several bugs in 4.2.3.
 Bugfixes
 ========
 
-* ...
+* Fixed a regression in Django 4.2 that caused a crash of
```

Comment:
```suggestion
* Fixed a regression in Django 4.2 that caused a crash in
```

Reply:
I would leave `of`.

Reply:
I don't mind changing but that was the wording used for ticket-34551

https://github.com/django/django/commit/e5c844d6f2a4ac6ae674d741b5f1fa2a688cedf4#diff-22f572de40cffa5d7430f4d645ff01556b2f5c870bdde06ec3bcb2ae305fef7cR36

Reply:
No worries then, resolving the comment as is.